pkg/didcomm/protocol/exchange/exchange_test.go

```diff
@@ -9,9 +9,13 @@ package exchange
 import (
 	"testing"
 
+	mocktransport "github.com/hyperledger/aries-framework-go/pkg/didcomm/transport/mock"
 	"github.com/stretchr/testify/require"
 )
 
+const destinationURL = "https://localhost:8090"
```

Comment:
this could be

const (
)

Reply:
Updated https://github.com/hyperledger/aries-framework-go/compare/c940bd1b350c2339769d648b80dc9fc401207265..1446f2c0373b23d55b26c068d9e9989fca4b8681#diff-bb59c2f695a7c208c1b2ff8067d70b4aR16